ReactiveCocoa/NSObject+KeyValueObserving.swift

```diff
@@ -53,6 +53,107 @@ extension Reactive where Base: NSObject {
 			signalLifetime += lifetime.observeEnded(observer.sendCompleted)
 		}
 	}
+
+	#if swift(>=3.2)
```

Comment:
Do we still support anything before this?

Reply:
No.